ansible/roles/alertmanager/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: create Alertmanager group
```

Comment:
IMHO, it should be enforced only for _handlers_ and other tasks should have `name` written with preference given to _readability_.